test/scripts/e2e_subs/tealprogs/app-abi-method-example.teal

```diff
@@ -1,68 +1,76 @@
-// generated from https://gist.github.com/jasonpaulos/99e4f8a75f2fc2ec9b8073c064530359
+// generated from https://gist.github.com/jasonpaulos/99e4f8a75f2fc2ec9b8073c064530359/b4d37519ccc67383042b6c0fb8b7b26a2e538738
```

Comment:
The PyTeal diff is here: https://gist.github.com/jasonpaulos/99e4f8a75f2fc2ec9b8073c064530359/revisions#diff-7f2d296a873a322392d91f69eebfc6993a857884a7e55cef9af800a408ef378f